README.md

```diff
@@ -75,30 +85,137 @@ Place the following in your `/etc/apache2/extra/httpd-vhosts.conf` file (replaci
 
 Make sure the following lines are enabled (uncommented) in your `/etc/apache2/httpd.conf` (cf. http://stackoverflow.com/a/1997047):
 ```
-    LoadModule proxy_module /usr/lib/apache2/modules/mod_proxy.so
-    LoadModule proxy_http_module /usr/lib/apache2/modules/mod_proxy_http.so
+LoadModule proxy_module /usr/lib/apache2/modules/mod_proxy.so
+LoadModule proxy_http_module /usr/lib/apache2/modules/mod_proxy_http.so
 
-    Include /private/etc/apache2/extra/httpd-vhosts.conf
+Include /private/etc/apache2/extra/httpd-vhosts.conf
 ```
 
 You may also need to add a `ServerName` declaration to `/etc/apache2/httpd.conf` to avoid warnings on Mac OS X:
-    ```ServerName localhost```
+```
+ServerName localhost
+```
 
-Add a line like the following to your /etc/hosts file:
-    ```127.0.0.1 localhost sc.local.concord.org```
+Add a line like the following to your `/etc/hosts` file:
+```
+127.0.0.1 localhost sc.local.concord.org
+```
 
 Restart apache so that the changes take effect:
-    ```apachectl -k graceful```
+```
+apachectl -k graceful
+```
 
-## Running Geniverse
+## Running Geniverse Locally
 
 Now run the development Geniverse-SproutCore server:
 ```
-    $ sc-server
+sc-server
 ```
   
 And then open http://sc.local.concord.org/lab in your browser.
 
+If you are unable to log in successfully, try opening a separate browser tab to http://learn-staging.concord.org and logging in manually there.
+
+## Deploying Geniverse
+
+### Installing and Configuring s3cmd Tools
+
+Download `s3cmd` from http://s3tools.org/download and follow the included installation instructions.
+
+Run `s3cmd --configure` to configure it. You will need your AWS account credentials (Access Key and Secret Access Key).
+
+### Installing Gems
```

Comment:
Any reason why this is in the Deploying section? Won't they already have encountered this when running?